CASMCMS-8672: ARM image sat bootprep documentation (#3958)
Cray-HPE/docs-csm: operations/iuf/stages/prepare_images.md

| Site variables                                | `-sv SITE_VARS`                  | Path to YAML file containing site defaults and any overrides                                          |
| Recipe variables product mask                 | `-mrp MASK_RECIPE_PRODS`         | Mask the recipe variables file entries for the products specified, use product catalog values instead |

> ### ARM Images
> 
> If the customer requires to build `aarch64` images see the [IUF `sat bootprep` ARM](../IUF.md#arm-images)

Reviewer: ```suggestion
### ARM images

If it is necessary to build `aarch64` images, then see [ARM images](../IUF.md#arm-images).
```
